tests/components/light/test_mqtt.py

```diff
@@ -228,7 +228,7 @@ def test_controlling_state_via_topic(self): \
         self.assertIsNone(state.attributes.get('effect'))
         self.assertIsNone(state.attributes.get('white_value'))
         self.assertIsNone(state.attributes.get('xy_color'))
-        self.assertIsNone(state.attributes.get(ATTR_ASSUMED_STATE))
+        self.assertFalse(state.attributes.get(ATTR_ASSUMED_STATE))
```

Comment:
@balloob done. The problem is, if a device support assume_state it will be present but on a device without that implement it will not present. I think that is correct.